### ISSUE TITLE
create_database_dump error

### ISSUE THREAD

**Issue by ftgo** [tendenci/tendenci#477]
I am trying to create a database dump on '/explorer/export_database/'. I get the following messages (both xml and json): 

`Form submitted is valid!
usage: manage.py create_database_dump [-h] [--version] [-v {0,1,2,3}]
                                      [--settings SETTINGS]
                                      [--pythonpath PYTHONPATH] [--traceback]
                                      [--no-color]
manage.py create_database_dump: error: unrecognized arguments: 36 json 5`

Django 1.8.8; 
Tendenci: 7.1.381

**Comment by eschipul** [comment]
@ftgo - what version of postgres? (should be 9.3 if on T7.1x)
what version of Ubuntu server? (should be 14.04.2+)

On a side note most of the commands live here:
https://github.com/tendenci/tendenci/tree/master/tendenci/apps/base/management/commands
and this is the explorer extension that generates the db dump.

tracking that back I see we are behind our our e-pantry django explorer module. The error is also seen here:
 http://stackoverflow.com/questions/30230490/django-custom-command-error-unrecognized-arguments

In other words we need to update Tendenci with thelasion of
https://github.com/epantry/django-sql-explorer
----------

#### definite bug - temp work around would be to ssh into the server and do a manual db backup until we can get patched.

**Comment by eschipul** [comment]
SSH temp workaround:
-------

    sudo su
    # switch to postgres user 
    su postgres
    # note this must be copied to /tmp/ because postgres doesn't have "w" rights elsewhere
    pg_dump <DBNAME> > /tmp/<DBNAME>.pgsql
    # switch back to sudo
    exit

**Comment by ftgo** [comment]
thanks @eschipul. I am looking forward to see this issue fixed.

**Comment by eschipul** [comment]
Agreed. We always program imports and exports first as it differentiates a truly OPEN product where you don't have to beg some company to give you your own data! I write about it a lot. I'm sure this was introduced with the push to pip of tendenci 7.1 on django 1.8 as "explorer" was previously on django 1.7. Leaving this ticket open for tracking.

Example blog posts and help files on various import export options from a google search:
https://www.google.com/#q=tendenci+exports

**Comment by jennyq** [comment]
Resolved https://github.com/tendenci/tendenci/commit/cc1c99738f07f05406e9c4ecb97b576c80c0661f

Also updated tendenci with the latest version of django-sql-explorer 0.9.2.
